search/tests/tests.py

```diff
@@ -604,12 +574,14 @@ def test_pagination(self):
 
     def test_exclude_ids(self):
         """ Test that ids that would normally be present in the resultset will not be present if in the exclude list """
-        self.searcher.index("test_doc", [{"course": "ABC", "id": "FAKE_ID_1"}])
-        self.searcher.index("test_doc", [{"course": "ABC", "id": "FAKE_ID_2"}])
-        self.searcher.index("test_doc", [{"course": "ABC", "id": "FAKE_ID_3"}])
-        self.searcher.index("test_doc", [{"course": "XYZ", "id": "FAKE_ID_11"}])
-        self.searcher.index("test_doc", [{"course": "XYZ", "id": "FAKE_ID_12"}])
-        self.searcher.index("test_doc", [{"course": "XYZ", "id": "FAKE_ID_13"}])
+        self.searcher.index([
+            {"course": "ABC", "id": "FAKE_ID_1"},
+            {"course": "ABC", "id": "FAKE_ID_2"},
+            {"course": "ABC", "id": "FAKE_ID_3"},
+            {"course": "XYZ", "id": "FAKE_ID_11"},
+            {"course": "XYZ", "id": "FAKE_ID_12"},
+            {"course": "XYZ", "id": "FAKE_ID_13"}
```

Comment:
Here do not forget about comma in the end

Reply:
Done